File: 2974-minimum-number-game/2974-minimum-number-game.java
class Solution {
    public int[] numberGame(int[] nums) {
       Arrays.sort(nums);
       int f=0;
       int b=1;
       while(b<=nums.length-1){
        int temp=nums[b];
        nums[b]=nums[f];
        nums[f]=temp;
        f+=2;
        b+=2;
       }
       return nums;

    }
}

File: 2974-minimum-number-game/2974-minimum-number-game.cpp
class Solution {
public:
    vector<int> numberGame(vector<int>& nums) {
        sort(nums.begin(),nums.end());
        int n=nums.size();
        for(int i=0;i<n-1;i+=2){
            int tmp=nums[i];
            nums[i]=nums[i+1];
            nums[i+1]=tmp;
        }
        return nums;
    }
};